hbase-server/src/test/java/org/apache/hadoop/hbase/quotas/TestQuotaAdmin.java

```diff
@@ -994,4 +994,16 @@ public int getQuotaSettingCount(Admin admin) throws IOException {
     }
     return quotaSettingCount;
   }
+
+  @Test
+  public void testQuotaTableDisableAndEnable() throws Exception {
+    final Admin admin = TEST_UTIL.getAdmin();
+    admin.disableTable(QuotaUtil.QUOTA_TABLE_NAME);
+    try {
+      admin.enableTable(QuotaUtil.QUOTA_TABLE_NAME);
+    } catch (Exception ex) {
+      LOG.error("", ex);
```

Comment:
Error message is empty and we have assertion message, se we may dont need a log message here.

Reply:
done